### PR TITLE
Scripts for digg tcl migration to balancer/aura eco

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #655 

To run in this order: 

```
brownie run issue/655/digg_tcl_migration graviaura_transfer

brownie run issue/655/digg_tcl_migration migrate
```

Pushing it to have ready for the treasury decision output if ✅ as discussed during call, nice if post as soon as it is possible

### CODE REVIEW

File: scripts/issue/655/digg_tcl_migration.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    # remove sushi tcl
+    vault.sushi.remove_liquidity(slp, slp.balanceOf(vault))
+
+    # claim aura rewards
+    vault.aura.claim_all_from_booster()
+
+    # deposit in graviaura and sent BAL claim to voter
+    aura.approve(graviaura, aura.balanceOf(vault))
+    graviaura.deposit(aura.balanceOf(vault))
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/issue/655/digg_tcl_migration.py]: good idea, push suggestion in latest commit

File: scripts/issue/655/digg_tcl_migration.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+def graviaura_transfer():
+    voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+    graviaura = voter.contract(r.sett_vaults.graviAURA)
+
+    graviaura.transfer(
+        r.badger_wallets.treasury_vault_multisig, graviaura.balanceOf(voter)
+    )
+
+    voter.post_safe_tx()
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/issue/655/digg_tcl_migration.py]: makes sense, pushed

**Review by gosuto-inzasheru — APPROVED** [review]
i suggest to run this after #642 in order to maximise the amount of (gravi)aura available

File: scripts/issue/655/digg_tcl_migration.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    # remove sushi tcl
+    vault.sushi.remove_liquidity(slp, slp.balanceOf(vault))
+
+    # claim aura rewards
+    vault.aura.claim_all_from_booster()
+
+    # deposit in graviaura and sent BAL claim to voter
+    aura.approve(graviaura, aura.balanceOf(vault))
+    graviaura.deposit(aura.balanceOf(vault))
+    bal.transfer(r.badger_wallets.treasury_voter_multisig, bal.balanceOf(vault))
```
[the remark this answers is not in the document]
Comment (sajanrajdev) [reply scripts/issue/655/digg_tcl_migration.py:42]: See the same and it doesn't make sense, looked at the events and there are no more underlying income transfers of BAL after this (as expected).

Maybe something off with brownie's fork env and accounting.

**Review by sajanrajdev — APPROVED** [review]
- Removes all liquidity from DIGG/WBTC SLP
- Claims all Aura rewards: deposits all AURA into graviAURA and transfers all BAL to the voter multi
- Calculates mantissas for LP into Digg's balancer pool - (Total LP value at current prices: $161241.378255, proportions match the 40/40/20 distribution)
image
- Deposits and stake wBTC, DIGG and graviAURA for these amounts
- Stakes BPT on Aura with balance checks - LG

Second script: simple transfer calls from the voter multi to the vault for graviAURA and AURA

LGTM

**Pull request merged**